FrontEnd/src/components/SearchPage/Search.js

```diff
@@ -0,0 +1,109 @@
+import './search_page.css';
+import frame42 from './img/frame42.png';
+import React, { useState, useEffect } from 'react';
```

Comment:
Same, no need to import `React`. Usually, its at the top

Reply:
Fixed